data/users/network-test/src/commonMain/kotlin/dev/burnoo/demo/listapp/data/users/network/test/GetUserResponse.kt

```diff
@@ -0,0 +1,24 @@
+package dev.burnoo.demo.listapp.data.users.network.test
+
+fun getUserResponse() = """
+{
+    "id": "${TestApiUser.id}",
+    "title": "${TestApiUser.title}",
+    "firstName": "${TestApiUser.firstName}",
+    "lastName": "${TestApiUser.lastName}",
+    "picture": "${TestApiUser.photoUrl}",
+    "gender": "${TestApiUser.gender}",
+    "email": "${TestApiUser.email}",
+    "dateOfBirth": "1952-09-03T13:27:29.424Z",
+    "phone": "${TestApiUser.phone}",
+    "location": {
+        "street": "8750, Rua Carlos Gomes",
+        "city": "Recife",
+        "state": "CearÃ¡",
+        "country": "Brazil",
+        "timezone": "+1:00"
+    },
+    "registerDate": "2021-06-21T21:02:07.719Z",
+    "updatedDate": "2021-06-21T21:02:07.719Z"
+}
+""".trimIndent()
```

Comment:
Another approach would be to use a serialization framework to generate the JSON from some model you already have (do you?). Less worrying about making a dumb mistake in the manually-written JSON, but it's mostly nitpicking.

Reply:
I like current approach as it is edited exact backend response, so I could update it when backend response changes and potentially find some backend errors.